src/foam/nanos/auth/User.js

```diff
@@ -173,7 +173,7 @@ foam.CLASS({
       documentation: 'User\' profile picture.',
       view: {
         class: 'foam.nanos.auth.ProfilePictureView',
-        placeholderImage: 'images/ic-placeholder.png'
+        placeholderImage: 'images/merchant/ic-placeholder.png'
```

Comment:
can we move the image please to images/ . 
merchant should be deprecated and everything moved.